rootfs/usr/bin/steamos-polkit-helpers/steamos-priv-write

```diff
@@ -13,15 +13,14 @@ WRITE_VALUE="$2"
 function CommitWrite()
 {
     echo "commit: $WRITE_VALUE -> $WRITE_PATH" | systemd-cat -t p-steamos-priv-write -p warning
-    echo "$WRITE_VALUE" > "$WRITE_PATH"
     chmod a+w "$WRITE_PATH"
```

Comment:
what if $WRITE_PATH doesn't exist? `set -e` means the script would error out at this point.

Reply:
Write path should be a sysfs path. CommitWrite is only called if if matches a case from below in the file. I would expect it to fail if the path is missing.